pkg/resources/k8sgpt.go

```diff
@@ -266,8 +266,15 @@ func GetDeployment(config v1alpha1.K8sGPT, outOfClusterMode bool, c client.Clien
 			addRemoteCacheEnvVar("AZURE_TENANT_ID", "azure_tenant_id")
 			addRemoteCacheEnvVar("AZURE_CLIENT_SECRET", "azure_client_secret")
 		} else if config.Spec.RemoteCache.S3 != nil {
-			addRemoteCacheEnvVar("AWS_ACCESS_KEY_ID", "aws_access_key_id")
```

Comment:
Why do we need different env variables although we are using the same aws client,  in theory this shouldn't matter if it's s3 native or minio?